sql/data_collection_helpers/bloated_tables_and_indexes.sql

```diff
@@ -123,16 +123,23 @@ DECLARE
       bloat_size::numeric,
       pg_size_pretty(bloat_size::int8)
     FROM (
-        SELECT current_database(), nspname AS schemaname, tblname, idxname, bs*(sub.relpages)::bigint AS real_size,
-        bs*est_pages::bigint as estimated_size,
-        bs*(sub.relpages-est_pages)::bigint AS bloat_size,
-        100 * (sub.relpages-est_pages)::float / sub.relpages AS bloat_ratio, is_na
+        SELECT current_database(), nspname AS schemaname, tblname, idxname, bs*(relpages-est_pages)::bigint AS extra_size,
```

Comment:
This is a good goal, although I propose to not do it in the same PR. The changes you propose will touch almost every line in the function, so it would be impossible to see which part of the query changed because of the update and which one due to the reformatting.